src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -2752,7 +2752,7 @@ enum Nf {
 
 #undef INSN
 
-  // RVB pseudo instructions
+  // Bit-manipulation extension pseudo instructions
   // zero extend word
   void zext_w(Register Rd, Register Rs) {
```

Comment:
Seems we can move `zext_w` to MacroAssembler too.

Reply:
Done. But the caller should be aware that this actually uses instructions from the Bit-manipulation extension. I think the code comment should make this explicit. Thanks.